package.json

```diff
@@ -20,7 +20,7 @@
     "gulp-uglify": "^1.5.3",
     "knuth-shuffle": "^1.0.1",
     "testem": "^1.0.2",
-    "typescript": "^2.0.6",
+    "typescript": "^2.1.0",
```

Comment:
Because TypeScript 2.1 is a breaking change from 2.0, I recommend we first publish the existing TypeScript-based snabbdom version, then later merge this PR and publish it as a breaking change (yes, bumping `major`).